Makefile

```diff
@@ -98,3 +98,30 @@ build-license-notice:
 .PHONY: generate-chart-docs
 build-chart-docs:
 	helm-docs -c charts/newrelic-prometheus-agent
+
+## Release Toolkit targets
+ifeq (, $(shell which rt))
+RT_BIN ?= go run github.com/newrelic/release-toolkit@latest
+else
+RT_BIN ?= rt
+endif
+
+.PHONY: release-notes
+release-notes: _release-changes
+	@$(RT_BIN) render-changelog
+	@echo "RELEASE NOTES:\n"
+	@cat CHANGELOG.partial.md
+
+### Upgrades the CHANGELOG.md as if a Release is being triggered.
+.PHONY: release-changelog
+release-changelog: _release-changes
+	@VER=$$($(RT_BIN) next-version --tag-prefix "v") && \
+	$(RT_BIN) update-markdown --markdown CHANGELOG.md --version $$VER
```

Comment:
I ended up in my PR with:
```
@$(RT_BIN) update-markdown --markdown CHANGELOG.md --version $$($(RT_BIN) next-version)
```

I find it more readable, but I am fine with both

Reply:
sure i can change it to that, I have no preferences